app/app.rb

```diff
@@ -0,0 +1,64 @@
+require 'sinatra/base'
+require "sinatra/json"
+require './lib/menu'
+require './lib/order_list'
+require './lib/order'
+require './lib/tax'
+require './lib/discount'
+require './lib/receipt'
+require './lib/payment'
+require './app/modules/my_helpers'
+class TillTechTest < Sinatra::Base
+  set :root, File.dirname(__FILE__)
+  helpers Sinatra::JSON
+  include MyHelpers
+  set :static, true
+  set :order_list, OrderList.new
+  set :menu,       Menu.new([{ name: 'Cafe Latte', price: 2.5 },
+                             { name: 'Cappucino',  price: 3.0 },
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.